.husky/pre-commit

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env sh
 . "$(dirname -- "$0")/_/husky.sh"
 
-yarn lint-staged
+npx lint-staged
```

Comment:
We should keep using yarn as the main dependency version as we have one `yarn.lock`.
```suggestion
yarn lint-staged
```

Reply:
The husky documentation recomends to use npx and this configuration doesn't affect nothing, but I'll change this